dts/bindings/timer/nuclei,machine-timer.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuclei RISC-V Machine timer
+
+compatible: "nuclei,machine-timer"
+
+include: base.yaml
+
+properties:
+    clk-divider:
+      type: int
+      required: true
+      description: |
+        clk-divider specifies the division ratio to the CPU frequency that
+        clock used by machine timer.
+        This property supports the case that the machine timer and CPU use
+        different clock sources.
+        This configuration is used sometimes for such as low power consumption.
+
+        For example, the CPU clock frequency is 108MHz, and the machine timer
+        uses 27MHz, which is the CPU clock divided by 4.
+        In this case, the CPU clock frequency is defined in the CPU node as follows
+
+           clock-frequency = <108000000>;
+
+
+        This property takes exponent of the power of 2.
+        The relationship with the frequency division ratio is as following equation.
```

Comment:
Keep documentation inside 80 columns.
Drop 1 line after clock-frequency.

Reply:
I'll correct it.